components/admin_console/team_channel_settings/team/details/team_profile.tsx

```diff
@@ -201,31 +249,49 @@ export function TeamProfile({team, isArchived, onToggleArchive, isDisabled, save
                                 {team.description || <span className='greyed-out'>{localizeMessage('admin.team_settings.team_detail.profileNoDescription', 'No team description added.')}</span>}
                             </div>
                         </div>
+                        <div className='team-desc-col'>
+                            <div className='row-bottom-padding'>
+                                <FormattedMarkdownMessage
+                                    id='admin.team_settings.team_detail.teamId'
+                                    defaultMessage='**Team ID**:'
+                                />
+                                <br/>
+                                {team.id}
+                            </div>
+                        </div>
                     </div>
                     <div className='AdminChannelDetails_archiveContainer'>
-                        {button()}
-                        {restoreDisabled &&
-                            <button
-                                onClick={() => {
-                                    dispatch(openModal({
-                                        modalId: ModalIdentifiers.PRICING_MODAL,
-                                        dialogType: PricingModal,
-                                    }));
-                                }}
-                                type='button'
-                                className={
-                                    classNames(
-                                        'btn',
-                                        'btn-secondary',
-                                        'upgrade-options-button',
-                                    )
-                                }
-                            >
-                                <FormattedMessage
-                                    id={'workspace_limits.teams_limit_reached.view_upgrade_options'}
-                                    defaultMessage={'View upgrade options'}
-                                />
-                            </button>}
+                        <div
+                            className='d-flex flex-row'
+                            style={{
+                                gap: '1rem',
+                            }}
+                        >
+                            {team && team.invite_id && <CopyLinkButton team={team}/> }
```

Comment:
I don't love this check here, but the tests will fail if the `invite_id` doesn't exist.

Reply:
With a slightly misleading error

```
   Unexpected console logsError: Uncaught [TypeError: Cannot read properties of undefined (reading 'roles')]

      37 | }
      38 | const CopyLinkButton = ({
    > 39 |     team,
         |     ^
      40 | }: CopyLinkButtonProps) => {
      41 |     const inviteURL = useMemo(() => {
      42 |         return `${getSiteURL()}/signup_user_complete/?id=${team.invite_id}&sbr=${getTrackFlowRole()}`;

      at reportException (node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:24)
      at innerInvokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:341:9)
      ```

Reply:
Yeah it's a bit misleading that `invite_id` isn't marked as optional in the types package when there are occasions where it's empty. Why is invite_id missing sometimes here?

Reply:
If i remember correctly, it's because in some of the tests we instantiate the test before the team is rendered so it fails. The check here just prevents that from happening.